packages/ui/Drawing/Drawing.js

```diff
@@ -28,19 +40,96 @@ const drawing = (beginPoint, controlPoint, endPoint, contextRef, isErasing) => {
 	contextRef.current.closePath();
 };
 
+/**
+ * A basic drawing canvas component.
+ *
+ * @class DrawingBase
+ * @memberof ui/Drawing
+ * @ui
+ * @public
+ */
 const DrawingBase = kind({
-	name: 'Drawing',
+	name: 'ui:Drawing',
 
 	functional: true,
 
-	propTypes: {
+	propTypes: /** @lends ui/Drawing.DrawingBase.prototype */ {
+
+		/**
+		 * Indicates the color of the brush.
+		 *
+		 * @type {String}
+		 * @default 'green'
+		 * @public
+		 */
 		brushColor: PropTypes.string,
+
+		/**
+		 * Indicates the size of the brush.
+		 *
+		 * @type {String}
```

Comment:
type should be `Number`
```suggestion
		 * @type {Number}
```